src/import-csv.js

```diff
@@ -0,0 +1,56 @@
+import fs from 'fs';
+import Promise from 'bluebird';
+import CouchBulkImporter from 'couchbulkimporter';
+import parse from 'csv-parse';
+import BulkBadger from 'bulkbadger';
+
+export function cli (file, url, ...csvOptions) {
+  if (!file) {
```

Comment:
idea is to avoid these types of errors which will confuse the user:

```
(13:19:42) [robert@tequila-work] ~/apache/couchdb-nmo (pr/1 *) $ ./bin/nmo-cli.js import-csv sddf sdf dsf
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: ENOENT, open 'sddf'
    at Error (native)
```